src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/Manifest.cs

```diff
@@ -66,23 +66,10 @@ public void Save(Stream stream)
         /// <param name="minimumManifestVersion">The minimum manifest version that this class must use when saving.</param>
         public void Save(Stream stream, int minimumManifestVersion)
         {
-            Save(stream, validate: true, minimumManifestVersion: minimumManifestVersion);
-        }
-
-        public void Save(Stream stream, bool validate)
-        {
-            Save(stream, validate, minimumManifestVersion: 1);
-        }
 
-        public void Save(Stream stream, bool validate, int minimumManifestVersion)
-        {
-            if (validate)
-            {
-                // Validate before saving
-                Validate(this);
-            }
+            Validate(this);
```

Comment:
why make it mandatory that all saves validate?

Reply:
This is manifest metadata validation. 
Namely validation such as, make sure the version is correct, amke sure the ID is valid etc. 

There's also no place in the code that currently skips the validation. 
It's unreachable.

Reply:
I'm all for deleting unnecessary code. 💯